scripts/migrate_master_azs.sh

```diff
@@ -0,0 +1,152 @@
+#!/bin/bash
+
+set -e
```

Comment:
Maybe it would be a good idea to use `pipefail` so the script doesn't continue if some `yq` parsing fail.

Reply:
done